src/class/SimpleData.ts

```diff
@@ -727,6 +728,28 @@ export default class SimpleData {
         return this
     }
 
+    @logCall()
+    pickRandomItems({
+        numberOfRandomItems,
```

Comment:
To be coherent with other options names, and because we know they will be random, I suggest nbItems instead of numberOfRandomItems.

Reply:
You can also add a default value. I suggest 100.

Reply:
done in fbc1e4b84133626d49aa60b29266e44e9304b9a0 !